src/lib.rs

```diff
@@ -1619,6 +1619,61 @@ impl<B: BitBlock> BitVec<B> {
     pub fn shrink_to_fit(&mut self) {
         self.storage.shrink_to_fit();
     }
+
+    /// Inserts a given bit at index `at`, shifting all bits after by one
+    ///
+    /// # Panics
+    /// Panics if `at > nbits`
+    ///
+    /// # Examples
+    ///```
+    /// use bit_vec::BitVec;
+    ///
+    /// let mut b = BitVec::new();
+    ///
+    /// b.push(true);
+    /// b.push(true);
+    /// b.insert(1, false);
+    ///
+    /// assert_eq!(b.len(), 3);
+    /// assert!(b.eq_vec(&[true, false, true]));
+    ///```
+    ///
+    /// # Time complexity                                                                                                                                                         
+    /// Takes O([`BitVec::len`]) time. All items after the insertion index must be
```

Comment:
Please add a fragment specifier below

[\`BitVec::len\`]: Self::len

Reply:
Then I'm not sure why one of our contributors was adding fragment specifiers. Ah, it's because he was changing all stuff like `BitVec::len` to just `len`.

Reply:
@Jujumba you may change to just `len`